data-store/chapter.md

```diff
@@ -752,16 +728,15 @@ are just a sublcass of ``ValueRef``.
 Storing richer data
 (via [``json``](http://json.org)
 or [``msgpack``](http://msgpack.org))
-is just a matter of writing your own
+is a matter of writing your own
 and setting it as the ``value_ref_class``.
 ``BinaryNodeRef`` is an example of using
 [``pickle``](https://docs.python.org/3.4/library/pickle.html)
 to serialise data.
 
-Database compaction.
-Compacting should be as simple as
-doing an infix-of-median traversal of the tree
-writing things out as you go.
+Database compaction:
+Compacting can be done via an infix-of-median traversal of the tree writing
+things out as you go.
```

Comment:
It's still an exercise! :) It might be good here to conclude the paragraph with something vague like "there are some subtleties to this, which you may discover if you try to implement it yourself."